nixos/modules/services/web-apps/gerrit.nix

```diff
@@ -222,6 +222,27 @@ in
         StandardOutput = "journal";
         StateDirectory = "gerrit";
         WorkingDirectory = "%S/gerrit";
+        AmbientCapabilities = "";
+        CapabilityBoundingSet = "";
+        LockPersonality = true;
+        NoNewPrivileges = true;
+        PrivateDevices = true;
+        PrivateTmp = true;
+        ProtectClock = true;
+        ProtectControlGroups = true;
+        ProtectHome = true;
+        ProtectHostname = true;
+        ProtectKernelLogs = true;
+        ProtectKernelModules = true;
+        ProtectKernelTunables = true;
+        ProtectProc = "noaccess";
+        ProtectSystem = "full";
+        RestrictAddressFamilies = [ "AF_UNIX" "AF_INET" "AF_INET6" ];
+        RestrictNamespaces = true;
+        RestrictRealtime = true;
+        RestrictSUIDSGID = true;
```

Comment:
Does gerrit support sending mail via a `sendmail` executable, or only via an stmp server? I think this would break `sendmail` integration.

Reply:
Gerrit only supports sending mails over an SMTP server. There is no sendmail integration.